modules/shadcn-nuxt.yml

```diff
@@ -0,0 +1,19 @@
+name: shadcn-nuxt
+description: shadcn-vue module for Nuxt
+repo: radix-vue/shadcn-vue#dev
+npm: shadcn-vue
```

Comment:
#### expect

```suggestion
npm: shadcn-nuxt
```

#### actual

Fixed by autofix. No matter how many times I tried, the result was the same because I didn't have the authority to stop CI/CD.

Reply:
The repo needs to point to the path of the Nuxt module in order for the auto-sync to work

Reply:
@danielroe , thanks for your advice I got the intended result. 🙏